### PR TITLE
Remove configuration option and translate applinking documentation to english

### PR DESCRIPTION
**Pull request by guilhermebruzzi**

### CODE REVIEW

File: docs/acquirers.en.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+# App Linking
+
+This documentation explains how inStore's app execute it's AppLinking feature (android's Intent or RCTLinking on iOS) with the possible actions: `payment` and `payment-reversal` that are sent to the payment app that are integrated with inStore.
```
Comment (darlenedms) [inline docs/acquirers.en.md]: "**A**ndroid's Intent" *(capital A)*
"... that are sent to the payment app ~~that are~~ integrated with inStore." *(remove second "that are")*

File: docs/acquirers.en.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+## VTEX PCI Gateway
+
+To make the integration work, it's necessary to create an configuration on VTEX Gateway's admin (our payment backend), with all the extra information necessary to the transaction (like the acquirer affiliation id or token).
```
Comment (darlenedms) [inline docs/acquirers.en.md]: "...to create **a** configuration on..." *(should be "a" instead of "an")*

File: docs/acquirers.en.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+To create any extra configuration you just need to pass to the instore team (instoredevs@vtex.com.br) what are the extra params needed to your app do the transaction and we will create a form on VTEX Gateway so that the client can configure it.
+
```
Comment (darlenedms) [inline docs/acquirers.en.md]: "To create any extra configuration you just need to ~~pass~~ **send** to ~~the~~ inStore team (instoredevs@vtex.com.br) ~~what are~~ the extra params ~~needed to~~ your app **need** to **complete** the transaction and we will create a form on VTEX Gateway so ~~that~~ the client can configure it." *(rephrase)*

File: docs/acquirers.en.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+The AppLinking integration doesn't include any other dependency, since the communication between the inStore app and the payment app happens with specific URIs with all the configuration and payment params that are necessary to do the action.
+
+> Observation:
+>
+> On Android, all communication must happen with a new `Intent`, so you shouldn't send the response as a `callback` of the initial Intent, instead you should send a new `Intent` to inStore app with the response to the original `Intent`.
+
```
Comment (darlenedms) [inline docs/acquirers.en.md]: "..the initial `Intent`..." *(Since you are highlighting all `Intent` words, there is one missing.)*

And, do we need the last "`Intent`"? In think it could end up like "...to inStore app with the response to the original." or "...to inStore app with the response to the **first one**."

File: docs/acquirers.en.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+## Types of configuration fields
+
+* General (all apps have):
+  * acquirerProtocol: string (ex.: stone, cielo-lio, cappta, vtex-sitef, seeed etc.) - AppLinking Protocol (scheme of each payment app)
+  * scheme: string (The scheme of the app that the `Intent` with the response will call, that by default will be "instore")
```
Comment (darlenedms) [inline docs/acquirers.en.md]: I'm not sure about this one, I would suggest something like:
"The **app** scheme **payment apps** ~~of the app that the~~ `Intent`**s** ~~with the response~~ will ~~call~~ **respond to**, ~~that~~ by default **it** will be "instore"."
*result:*
"The app scheme payment apps `Intent`s will respond to, by default it will be "instore"."

File: docs/acquirers.en.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+  * autoConfirm (by default its value is "true". Can be used on the payment app to know that the app doesn't need to ask to the user any other permission to do the action)
```
Comment (darlenedms) [inline docs/acquirers.en.md]: Maybe rephrase to "Indicates to the payment app it doesn't need to ask any additional permission to the user in order to complete the action."

File: docs/acquirers.en.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+  * acquirerId: string (ex.:<stone_code>, <sitef_storeId>) affiliation id of the acquirer that is registered on VTEX Gateway
+
+If it's necessary, inStore can send additional information. Example with the acquirer Cappta:
+
```
Comment (darlenedms) [inline docs/acquirers.en.md]: Maybe rephrase "Example with the acquirer Cappta" to "Cappta acquirer example:"

File: docs/acquirers.en.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+  * authPassword: string (e.g. "<cappta_authPassword>")
+  * administrativePassword: string (with default password "cappta")
+  * cnpj: string
+
+## Sender URI and Response URI for each action
+
+URI pattern that is sent by the AppLinking:
+
+```
+<acquirerProtocol>://<action>?<parametros>
```
Comment (darlenedms) [inline docs/acquirers.en.md]: params

File: docs/acquirers.en.md [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+```
+
+URI pattern that is received by the AppLinking:
+
+```
+instore://<action>?<parametros_de_resposta>
```
Comment (darlenedms) [inline docs/acquirers.en.md]: response_params

File: docs/acquirers.en.md [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+
+### Examples of Sender URI for each action
+
+##### - Example for the action "payment":
+
+Context of payment that is used to assemble the URI (so that is easier to read):
+
+```
+{
+  acquirerProtocol: "super-adquirente",
```
Comment (darlenedms) [inline docs/acquirers.en.md]: super-acquirer

File: docs/acquirers.en.md [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+  action: "payment",
+  acquirerId: "954090369",
+  installmentType: 2,
+  installments: 3,
+  paymentId: "1093019888",
+  paymentType: "debit",  // could be credit also
+  amount: 10, // as apps esperam o valor em centavos (10 centavos)
+  installmentsInterestRate: "1%", (se não tiver juros, então não é nem para estar no mobileLinkingUrl)
```
Comment (darlenedms) [inline docs/acquirers.en.md]: these comments need to be translated:
"// payment apps usually expect the amount in cents (10 cents)"
"// (if the order doesn't have interest rate it shouldn't be on `mobileLinkingUrl`)"

File: docs/acquirers.en.md [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+  transactionId: "1093019039",
+  scheme: "instore",
+  autoConfirm: "true",
+  paymentSystem: 44,
+  paymentSystemName: "Venda Direta Debito",
```
Comment (darlenedms) [inline docs/acquirers.en.md]: I believe it can't be translated, right?

File: docs/acquirers.en.md [hunk 13]
```diff
@@ -0,0 +1,160 @@
...
+  paymentGroupName: "debitDirectSalePaymentGroup",
+  mobileLinkingUrl: "super-adquirente://payment?acquirerId=954090369&paymentId=1093019888&paymentType=debit&amount=10&installments=3&transactionId=1093019039&autoConfirm=true&scheme=instore"
```
Comment (darlenedms) [inline docs/acquirers.en.md]: super-acquirer

File: docs/acquirers.en.md [hunk 14]
```diff
@@ -0,0 +1,160 @@
...
+}
+```
+
+Final URI with the previous context that the payment app will receive to do the payment:
```
Comment (darlenedms) [inline docs/acquirers.en.md]: Maybe
"Final URI with the previous context ~~that the~~ payment app will receive **in order** to **execute** ~~do~~ the payment **action**:"

File: docs/acquirers.en.md [hunk 15]
```diff
@@ -0,0 +1,160 @@
...
+
+```
+super-adquirente://payment?acquirerProtocol=super-adquirente&action=payment&acquirerId=954090369&installmentType=2&installments=3&paymentId=1093019888&paymentType=debit&amount=10&installmentsInterestRate=1%&transactionId=1093019039&paymentSystem=44&paymentSystemName=Venda%20Direta%20Debito&paymentGroupName=debitDirectSalePaymentGroup&scheme=instore&autoConfirm=true
```
Comment (darlenedms) [inline docs/acquirers.en.md]: super-acquirer

File: docs/acquirers.en.md [hunk 16]
```diff
@@ -0,0 +1,160 @@
...
+With the response of this AppLinking, will be possible to refund this payment.
+
+
+##### - Example for the action "payment-reversal" (refund):
+
+Context of the refund that is used to assemble the URI (so that is easier to read):
+
+```
+{
+  acquirerProtocol: "super-adquirente",
```
Comment (darlenedms) [inline docs/acquirers.en.md]: super-acquirer?

File: docs/acquirers.en.md [hunk 17]
```diff
@@ -0,0 +1,160 @@
...
+  action: "payment-reversal",
+  acquirerId: "954090369",
+  transactionId: "1093019039",
+  paymentId: "1093019888",
+  acquirerTid: "1093019888",
+  administrativeCode: "11010103033", // This value is expected to be received from the payment and is saved on VTEX Gateway
+  autoConfirm: "true",
+  scheme: "instore",
+  mobileLinkingUrl: "super-adquirente://payment-reversal?acquirerId=954090369&paymentId=1093019888&transactionId=1093019039&autoConfirm=true&scheme=instore"
```
Comment (darlenedms) [inline docs/acquirers.en.md]: super-acquirer

File: docs/acquirers.en.md [hunk 18]
```diff
@@ -0,0 +1,160 @@
...
+}
+```
+
+Final URI with the previous context that the payment app will receive to do the refund:
```
Comment (darlenedms) [inline docs/acquirers.en.md]: Maybe
"Final URI with the previous context ~~that the~~ payment app will receive **in order** to **execute** the refund **action**:"

File: docs/acquirers.en.md [hunk 19]
```diff
@@ -0,0 +1,160 @@
...
+
+```
+super-adquirente://payment-reversal?acquirerId=954090369&transactionId=1093019039&paymentId=1093019888&acquirerTid=1093019888&administrativeCode=11010103033&autoConfirm=true&scheme=instore
```
Comment (darlenedms) [inline docs/acquirers.en.md]: super-acquirer

File: docs/acquirers.en.md [hunk 20]
```diff
@@ -0,0 +1,160 @@
...
+
+
+### Examples of Response URI for each action
+
+##### - Example of a response for the action "payment":
+
+URI:
+
+```
+Success: instore://payment?responsecode=0&<parametros_de_resposta>
```
Comment (darlenedms) [inline docs/acquirers.en.md]: response_params

File: docs/acquirers.en.md [hunk 21]
```diff
@@ -0,0 +1,160 @@
...
+Failed:   instore://payment?responsecode=110&reason=erro+no+cartao+cancelado+pelo+cliente&paymentId=<value_of_the_sender_URI>
```
Comment (darlenedms) [inline docs/acquirers.en.md]: Will we translate these error messages (reason)?

File: docs/acquirers.en.md [hunk 22]
```diff
@@ -0,0 +1,160 @@
...
+```
+
+Response parameters:
+  * scheme: "instore"
+  * action: "payment"
+  * paymentId: string (identification of the payment on VTEX)
+  * cardBrandName: string (name of the card brand, like "mastercard", "visa", etc.)
+  * acquirerName: string (name of the acquirer - optional)
+  * acquirerTid: string (identification of the payment on the acquirer)
+  * acquirerAuthorizationCode or administrativeCode: string (authorization code that the app will receive on a refund action to authorize it)
```
Comment (darlenedms) [inline docs/acquirers.en.md]: Maybe rephrase to "string (authorization code **needed** to authorize a refund action)"

File: docs/acquirers.en.md [hunk 23]
```diff
@@ -0,0 +1,160 @@
...
+  * reason: string (in case of success it's value is empty and in case of an error this contains an error message)
+  * success: true or false (generated by the app given the value on responsecode)
+
+
+##### - Example of a response for the action "payment-reversal" (refund):
+
+URI:
+
+```
+Success: instore://payment-reversal?responsecode=0&<parametros_de_resposta>
```
Comment (darlenedms) [inline docs/acquirers.en.md]: response_params

File: docs/acquirers.en.md [hunk 24]
```diff
@@ -0,0 +1,160 @@
...
+Failed:   instore://payment-reversal?responsecode=110&reason=erro+no+cartao+cancelado+pelo+cliente&paymentId=<valor_enviado_na_ida>
```
Comment (darlenedms) [inline docs/acquirers.en.md]: Will we translate these error messages (reason)?

I don't know how to translate "valor_enviado_na_ida", maybe "id_already_sent"

File: docs/acquirers.en.md [hunk 25]
```diff
@@ -0,0 +1,160 @@
...
+```
+
+Response parameters:
+  * scheme: "instore"
+  * action: "payment-reversal"
+  * paymentId: string (e.g. "1093019888") to identify what payment was refunded
+  * acquirerAuthorizationCode or administrativeCode: string (authorization code that an acquirer can also give to an refund)
```
Comment (darlenedms) [inline docs/acquirers.en.md]: "... give to **a** ~~an~~ refund"

File: docs/acquirers.en.md (hunk 1 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 2 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 3 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 4 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 6 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 7 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: Coloquei: "acquirer Cappta example", me pareceu mais correto! :D

File: docs/acquirers.en.md (hunk 8 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 9 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 10 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 11 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 12 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: right! :(

File: docs/acquirers.en.md (hunk 13 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 14 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: Só mantive a parte do "context that the payment app", pq sem esse "that the", eu achei que a frase ficou estranha! :D

File: docs/acquirers.en.md (hunk 15 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 16 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 17 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 18 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done (with "that the")

File: docs/acquirers.en.md (hunk 19 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 20 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 21 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: Faz sentido, coloquei: "card+refused+by+acquire"

File: docs/acquirers.en.md (hunk 22 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 23 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 24 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done

File: docs/acquirers.en.md (hunk 25 above)
Comment (guilhermebruzzi) [reply docs/acquirers.en.md]: done
